libs/category/state/src/facades/category-facade.interface.ts

```diff
@@ -26,7 +27,16 @@ export interface DaffCategoryFacadeInterface<
   /**
    * The page configuration state for the selected category.
    */
-  pageConfigurationState$: Observable<DaffCategoryPageConfigurationState>;
+  pageConfigurationState$: Observable<DaffStatefulCategoryPageConfigurationState>;
+  /**
+   * The current loading state of the selected category page.
+   */
+  // TODO: fix this horrible overuse of "state"
```

Comment:
Maybe change the base model to `DaffCategoryPageConfiguration` in a future PR?

Reply:
it will be changed entirely later to a metadata model

Reply:
no clue what you mean, but I look forward to seeing it